### ISSUE TITLE
Git push --force question

### ISSUE THREAD

**Issue by elreydetoda** [chabad360/hugo-gh-pages#20]
Hello :slightly_smiling_face:  

First off  thank you for this project, it seems really cool :slightly_smiling_face:.

I was reading through the `docker-entrypoint.sh` and I was curious why you had used a `git push --force` instead of a `git push` and then allowed for an optional force through a configuration option?

https://github.com/chabad360/hugo-gh-pages/blob/0bcf754dc6c823a580c58a59a3baa476a41258d3/docker-entrypoint.sh#L76

**Comment by chabad360** [comment]
Hmm, that's an interesting point you bring up.

I don't remember why I made it that way, I did partially fork this from another project, so it's possible that line was already there. I believe the reason for it is "better safe than sorry" more than anything else. 

Frankly, I don't really see a reason why this should be optional, if you're worried that it might overwrite changes made to the site, that would be silly because you're using a static site generator, you shouldn't be modifying the results.
